src/methods/utility.js

```diff
@@ -0,0 +1,13 @@
+export const objToQueryStr = (obj) => isNotNil(obj) && isObject(obj)
+  ? '?' + Object.keys(obj).map(key => key + '=' + obj[key]).join('&')
+  : '';
+
+export const isNil = (val) => [null, undefined].includes(val);
+
+export const isNull = (val) => val === null;
+
+export const isNotNil = (val) => !isNil(val);
+
+export const isNotNull = (val) => !isNull(val);
+
+export const isObject = (val) => typeof val === 'object' && isNotNull(val);
```

Comment:
I believe this code needs to be wrapped in module.exports, etc.

Reply:
I convert this with module.exports